.terraform-docs.yml

```diff
@@ -9,35 +9,32 @@ content: |-
   {{ .Requirements }}
   ## Sample Usage
   ```hcl
-  module "example" {
-    {{"\n"}}
-    {{"\t"}} source  = "github.com/Cyber4All/terraform-cyber4all-catalog//modules/<REPLACE_WITH_MODULE>?ref=v<REPLACE_WITH_VERSION>"
-    {{"\n"}}
+  terraform {
+  {{"\t"}} source = "github.com/Cyber4All/terraform-cyber4all-catalog//modules/<REPLACE_WITH_MODULE>?ref=v<REPLACE_WITH_VERSION>"
+  }
 
+  inputs = {
```

Comment:
Lol this was all we needed to fix the inputs error?

Reply:
Yep its below